src/MCPClient/lib/clientScripts/archivematicaCreateMETS.py

```diff
@@ -174,6 +175,8 @@ def createFileSec(path, file_group_identifier, base_path, base_path_name, parent
                 createFileSec(os.path.join(path, item), file_group_identifier, base_path, base_path_name, parentBranch, div, sip_uuid)
 
 if __name__ == '__main__':
+    logger = get_script_logger("archivematica.mcp.client.createMETS")
```

Comment:
I'd like to keep a correlation between the script name and logger name, but let's look at renaming the scripts themselves in a PR after this is merged!